javascript/apps/taiga/src/app/modules/project/components/assign-user/assign-user.component.ts

```diff
@@ -333,8 +333,13 @@ export class AssignUserComponent implements OnInit, OnDestroy {
   public setCloseShortcut() {
     this.shortcutsService.setScope('assign-user');
     this.shortcutsService
-      .task('assign-user.close')
-      .pipe(untilDestroyed(this))
+      .task('cancel', {}, 'assign-user')
+      .pipe(
+        untilDestroyed(this),
+        finalize(() => {
+          this.shortcutsService.deleteScope('assign-user');
```

Comment:
If I open this in a modal, close the assign with ESC, I can't close the modal with ESC, probably you should use `undoLastScope`  or `undoScope`

Reply:
There is already a 'this.shortcutsService.undoLastScope();' in the ngOnDestroy so I delete the finalize.